lib/utils.js

```diff
@@ -1,6 +1,18 @@
 import NativePromise from 'native-promise-only'
+import pkgJson from '../package.json'
+
 const Promise = window.Promise || NativePromise
 
+export function addUserAgent (headers, appId) {
```

Comment:
You could add a test for this.

Reply:
Actioned with this commit https://github.com/zendesk/zendesk_app_framework_sdk/pull/220/commits/23e6df883bfab63cfa495c7c8bed55be8b603c91 , thanks for keeping my pull requests honest 😅